plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLSourceViewer.java

```diff
@@ -251,6 +255,23 @@ public void run() {
                         }
                     }, true));
             }
+            if (sourceObject.supportsObjectDefinitionOption(DBPScriptObject.OPTION_CAST_PARAMS)) {
```

Comment:
I don't see any DBPScriptObjectExt2 implementation which supportsObjectDefinitionOption for the new parameter. 
It should be supported by procedures I suppose? Or I miss something..

Reply:
@serge-rider , help me please - should I remove here

if (sourceObject.supportsObjectDefinitionOption(DBPScriptObject.OPTION_CAST_PARAMS)) {
                options.put(DBPScriptObject.OPTION_CAST_PARAMS, getShowCastParams());
            }

?
(I did it to be like other options in code)

Reply:
As I see the condition `sourceObject.supportsObjectDefinitionOption(DBPScriptObject.OPTION_CAST_PARAMS)` will never be true, because implementation of this method is like in PostgreProcedure:supportsObjectDefinitionOption
```
    public boolean supportsObjectDefinitionOption(String option) {
        return DBPScriptObject.OPTION_INCLUDE_COMMENTS.equals(option) || DBPScriptObject.OPTION_INCLUDE_PERMISSIONS.equals(option);
    }
```
I doubt how it works for you. I guess you need to modify `PostgreProcedure:supportsObjectDefinitionOption` like
```
    public boolean supportsObjectDefinitionOption(String option) {
        return DBPScriptObject.OPTION_INCLUDE_COMMENTS.equals(option) || DBPScriptObject.OPTION_INCLUDE_PERMISSIONS.equals(option) || DBPScriptObject.OPTION_CAST_PARAMS.equals(option);
    }
```

Reply:
Ahhh, thank you, @E1izabeth , I understand now